### PR TITLE
Deploy to VPS server

### PR DESCRIPTION
**Pull request by pastpatryk**

### CODE REVIEW

**Comment by glowacki-dev** [comment]
There are some links in readme as well

**Comment by pastpatryk** [comment]
@glowacki-dev changed the links in README 👍

File: .github/workflows/master.yml [hunk 1]
```diff
@@ -12,30 +12,38 @@ jobs:
...
-          project_id: octo-board
-          service_account_key: ${{ secrets.APP_ENGINE_SERVICE_ACCOUNT_OCTO_BOARD }}
-      - name: Deploy to App Engine
-        run: gcloud app deploy --quiet --version=${GITHUB_SHA}
+          SSH_PRIVATE_KEY: ${{secrets.SSH_PRIVATE_KEY}}
+          SSH_KNOWN_HOSTS: ${{secrets.SSH_KNOWN_HOSTS}}
+      - name: Create env file
+        run: |
+          touch .env
+          echo PRODUCTION_HOST=${{ secrets.PRODUCTION_HOST }} >> .env
```
Comment (glowacki-dev) [inline .github/workflows/master.yml:32]: Here, we're using secrets directly in run, but in a previous step we first assign them to env variables 👀

Comment (glowacki-dev) [reply .github/workflows/master.yml:32]: Yup that may be the case. I'd use env variables everywhere for consistency

**Review by glowacki-dev — APPROVED** [review]

**Pull request merged**